sdk/batch/azure-resourcemanager-batch/pom.xml

```diff
@@ -80,5 +80,11 @@
       <version>2.45.0</version> <!-- {x-version-update;com.azure.resourcemanager:azure-resourcemanager-storage;dependency} -->
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.azure.resourcemanager</groupId>
+      <artifactId>azure-resourcemanager-resources</artifactId>
+      <version>2.45.0</version> <!-- {x-version-update;com.azure.resourcemanager:azure-resourcemanager-resources;dependency} -->
+      <scope>test</scope>
+    </dependency>
```

Comment:
I see. You do not need to add resources, as `storage` lib already contains it.

Reply:
Fixed by [commit#421cec9](https://github.com/Azure/azure-sdk-for-java/pull/43422/commits/421cec9761e309fb53914653ddbc919b32f53355)